docs/csi_ug_install_operator_github.md

```diff
@@ -0,0 +1,50 @@
+# Installing the driver with GitHub
+
+The operator for IBM® block storage CSI driver can be installed directly with GitHub. Installing the CSI (Container Storage Interface) driver is part of the operator installation process.
+
+Use the following steps to install the operator and driver, with [GitHub](https://github.com/IBM/ibm-block-csi-operator) (github.com/IBM/ibm-block-csi-operator).
+1.  Download the manifest from GitHub.
+
+    ```
+    curl https://raw.githubusercontent.com/IBM/ibm-block-csi-operator/v1.6.0/deploy/installer/generated/ibm-block-csi-operator.yaml > ibm-block-csi-operator.yaml
+    ```
+
+2.  **Optional:** Update the image fields in the ibm-block-csi-operator.yaml.
+
+    **Note:** Updating the namespace to a user-defined namespace might be necessary to ensure consistency and avoid trouble with operator installation.
+
+3.  Create a project namespace, using the `kubectl create ns <namespace>` command.
+
+4.  Install the operator, while using a user-defined namespace.
+
+    ```
+    kubectl -n <namespace> apply -f ibm-block-csi-operator.yaml
+    ```
+
+5.  Verify that the operator is running. (make sure that the Status is _Running_.)
```

Comment:
is this [additional](https://github.com/IBM/ibm-block-csi-driver/blob/af456671de58029b42ece5e7b38d27cfc818123e/README.md#4-install-the-operator-while-using-a-user-defined-namespace) `5.` numbering on purpose? because I think it might prevent [upgrading users](https://github.com/IBM/ibm-block-csi-driver/pull/320/files#diff-1b6875ae24ff3eb04d63ec962eb0be5723d1976095835e0c745efc6d594339ddR14) from performing this step

Reply:
this was resolved as part of https://jira.xiv.ibm.com/browse/CSI-3077

Reply:
@rivkap please avoid using internal links on public GitHub

Reply:
@oriyarde I have to be able to link to the repository from both GitHub but also from IBM Docs. These are the same links that I've used in the past. Unless you have another link for me to use, this will stay as-is

Reply:
I was referring to the original version of [your comment here](https://github.com/IBM/ibm-block-csi-driver/pull/320#discussion_r650518372)